src/vs/workbench/browser/parts/titlebar/media/titlebarpart.css

```diff
@@ -24,8 +24,128 @@
 	flex: 0 1 auto;
 	overflow: hidden;
 	white-space: nowrap;
-	line-height: 22px;
```

Comment:
@sbatten why? this impacts even macOS that does not use a custom menu

Reply:
@bpasero the value is inherited from the parent and the parent is being updated to handle zoom